client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.html

```diff
@@ -19,26 +19,25 @@
                     data-translate-attr="{ title : 'xdsl_modem_device_connected' }"
                     type="button"
                     data-ng-click="DeviceCtrl.showDevices = !DeviceCtrl.showDevices">
-                <i class="ovh-font"
-                      data-ng-class="{
-                          'ovh-font-disclosure-right':!DeviceCtrl.showDevices,
-                          'ovh-font-disclosure-down':DeviceCtrl.showDevices}"
-                      aria-hidden="true">
-                </i>
-                <span data-ng-pluralize
-                      data-ng-if="DeviceCtrl.devices"
-                      data-count="DeviceCtrl.devices.length"
-                      data-when="{
-                          '0': ('xdsl_modem_device_show_none' | translate),
-                          'one': ('xdsl_modem_device_show_one' | translate : {count: DeviceCtrl.devices.length}),
-                          'other': ('xdsl_modem_device_show_many' | translate : {count: DeviceCtrl.devices.length})}">
+                <span data-ng-if="DeviceCtrl.devices.length">
+                    <span class="ovh-font"
```

Comment:
Don't we have an equivalent with UI Kit?

Reply:
We have chevron-right and down but they dont look same. We can take up bringing all components to new oui-kit standard as a new story in telecom